### DISCUSSION TITLE
How to navigate to the Trouble LSP References sidebar?

### DISCUSSION THREAD

**Discussion by pkazmier** [LazyVim/LazyVim#5119] [Q&A] [answered]
How do you navigate to the Troubke LSP References sidebar without its contents changing? I.e., if I use C-h/C-l/etc .. and I traverse another window containing a different file, then by time I make it to the sidebar, it has been updated based on the last window traversed. Of course, I could use a mouse, but who does that?

**Comment by dpetka2001** [comment]
You can pass to your keymap `focus=true`, so that your cursor automatically focuses the window.

**Reply by pkazmier** [reply]
I’m not sure I understand what you are saying. If you have two files open in separate windows side by side with cursor on a symbol on the leftmost window, then open the trouble LSP references sidebar on right side of screen, then want to navigate to one of the references without having to use your mouse to click on it, how would you do it? If you use C-l to move to the right and then C-l again to get to the sidebar, the contents of the sidebar have changed when the cursor traveled through the other window, right?

**Reply by dpetka2001** [reply, marked as answer]
Yes, that's correct. But if you change the default mapping then it will autofocus the Trouble window. Something like this
```lua
{
    "folke/trouble.nvim",
    keys = {
      { "<leader>cS", "<cmd>Trouble lsp toggle focus=true<cr>", desc = "LSP references/definitions/... (Trouble)" },
    },
},
```

**Reply by pkazmier** [reply]
Ah, I understand now. Thank you.
